public/posts/v1.6.md

```diff
@@ -0,0 +1,226 @@
+other than VSCode would require to duplicate the efforts all over. Providing a
+self-contained LSP implementation in Deno makes it easier to provide a
+high-quality and unified editing experience for all code editors that understand
+LSP.
+
+The LSP is in the early stages of the development, but it already supports some
+crucial features:
+
+- Go to definition
+- Go to references
+- Hints on hover
+- `deno fmt` integration
+- `deno lint` integration
+
+To track the progress of the development follow
+[issue #8643](https://github.com/denoland/deno/issues/8643).
+
+[VSCode Deno Canary](https://marketplace.visualstudio.com/items?itemName=denoland.vscode-deno-canary)
+is a new version of VSCode extension that uses `deno lsp`. It is also in the
+early stages of the development and is not yet as featureful as `VSCode Deno`.
+
+_If you decide to give it a try make sure to disable the old version of the
+extension, otherwise diagnostics might be duplicated. For now it is also
+recommended to use the Deno canary update channel when using the LSP, because
+updates and changes will still be frequent for the LSP._
+
+## Migration to stricter type checks complete
+
+In Deno 1.4 we introduced some stricter TypeScript type checks in `--unstable`
+that enabled us to move a bunch of code from JS into Rust (enabling huge
+performance increases in TypeScript transpilation, and bundling). In Deno 1.5
+these stricter type checks were enabled for everyone by default, with a opt-in
+in the form of the `"isolatedModules": false` TypeScript compiler option.
+
+In this release this override has been removed. All TypeScript code is now run
+with `"isolatedModules": true`.
+
+For more details on this, see the
+[Deno 1.5 blog post](https://deno.land/posts/v1.5#stricter-type-checks-in-stable).
+
+## TypeScript 4.1
+
+Deno 1.6 ships with the latest stable version of TypeScript.
+
+For more information on new features in Typescript 4.1 visit official
+introductory blog post:
+[Announcing TypeScript 4.1](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/)
+
+## Canary channel
+
+For advanced users that would like to test out bug fixes and features before
+they land in the next stable Deno release, we now provide a `canary` update
+channel. Canary releases are made multiple times a day, once per commit on the
+[master branch](https://github.com/denoland/deno/tree/master) of the Deno
+repository.
+
+You can identify these releases by the 7 character commit hash at the end of the
+version, and the `canary` string in the `deno --version` output.
+
+Starting with Deno 1.6, you can switch to the canary channel, and download the
+latest canary by running `deno upgrade --canary`. You can jump to a specific
+commit hash using
+`deno upgrade --canary --version 5eedcb6b8d471e487179ac66d7da9038279884df`.
+
+_Warning: jumping between canary versions, or downgrading to stable, may corrupt
+your DENO_DIR._
```

Comment:
This formatting is broken:
![Screenshot 2020-12-08 at 14 48 17](https://user-images.githubusercontent.com/13602871/101491755-6b05ff00-3964-11eb-8aaa-6487a2ccb8df.png)